locksmith/src/controllers/v2/metadataController.ts

```diff
@@ -235,10 +235,14 @@ export class MetadataController {
         return response.status(201).send(createdUserMetadata.data)
       }
 
-      return response.status(409).send('User Metadata already exists.')
+      return response.status(409).send({
+        message: "User metadata doesn't exist.",
+      })
```

Comment:
@searchableguy in this `409` case, the error message shouldn't be `[...] already exists`  ?

Reply:
I believe `409` is the correct HTTP status code for a resource that already exists and cannot perform updates. 

Do you have another suggestion?

Reply:
so we need can change it to `User Metadata already exists.` (as before) instead of `User metadata doesn't exist` 👍